tests/ts-react.test.js

```diff
@@ -71,7 +71,8 @@ test('preset development has hmr enabled for react', t => {
   const api = Neutrino();
   api.use(require('..'));
   const config = api.config.toConfig();
-  t.is(config.entry.index[0], join(api.options.root, 'node_modules\\react-hot-loader\\patch.js'));
+  const regexp = new RegExp(join(api.options.root, 'node_modules[\\/]react-hot-loader[\\/]patch.js'));
```

Comment:
Can you please explain a bit more about this change?
Would like to know how it effects the non-OS-specificity.